test/Tests/src/Data/Range_Spec.enso

```diff
@@ -56,4 +80,217 @@ spec = Test.group "Range" <|
         3.up_to 5 . contains 2 . should_be_false
         0.up_to 10 . contains -3 . should_be_false
 
+        0.up_to 10 . contains 2.5 . should_fail_with Illegal_Argument_Error
```

Comment:
what about is the `up_to` argument is a `Decimal`? It should also fail with Illegal Argument